store/tikv/2pc.go

```diff
@@ -1134,7 +1134,6 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 		}
 	}()
 
-	binlogPrewriteStart := time.Now()
 	binlogChan := c.prewriteBinlog(ctx)
```

Comment:
for a slow query, maybe we are more interested in the time used to wait for this channel at line 1151~?